service/start.go

```diff
@@ -162,7 +162,15 @@ func (svc *service) startAllExistingAppsWalletSubscriptions(ctx context.Context,
 
 	for _, app := range apps {
 		go func(app db.App) {
-			err := svc.startAppWalletSubscription(ctx, relay, *app.WalletPubkey)
+			// republish info event for all existing apps
+			walletPrivKey, err := svc.keys.GetAppWalletKey(app.ID)
```

Comment:
Don't we need a check `if app.WalletPubkey != nil {`?

Reply:
In the above SQL query we filter by the apps where `wallet_pubkey IS NOT NULL` right?

Reply:
Ah, right! thanks.